src/Symfony/Component/HttpKernel/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ CHANGELOG
 3.1.0
 -----
  * deprecated passing objects as URI attributes to the ESI and SSI renderers
+ * Added a `LegacyArgumentResolver` with `getArguments()` and the respective interface `ArgumentResolverInterface`
```

Comment:
@HeahDude my point is that the ``ArgumentResolverInterface`` should be mentionned as being the added feature, not the ``LegacyArgumentResolver``. So the interface should start the sentence, and ``LegacyArgumentResolver`` should be mentioned only later as an implementation